<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>山体地质灾害智能勘测系统登录界面</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{border-image:url(D:/procedure/groundstation/checkin/2.png)}</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Sans</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image:url(D:/procedure/groundstation/checkin/yellow.png)}</string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>135</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Sans</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>账号：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>230</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Sans</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image:url(D:/procedure/groundstation/checkin/yellow.png)}</string>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Sans</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
background-color:transparent;</string>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>-20</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(D:/procedure/groundstation/checkin/log.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Sans</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>山体地质灾害智能勘测系统</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background: white;
    selection-background-color: darkgray;

}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background: white;
    selection-background-color: darkgray;
}</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <zorder>label_4</zorder>
  <zorder>pushButton</zorder>
  <zorder>label</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>lineEdit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
